res/skins/Deere/style.qss

```diff
@@ -113,11 +113,17 @@
 
 #Spacer22 {
   background-color: #222222;
-}
+} /*
```

Comment:
delete commented out code

Reply:
hmm I'll uncomment it then. It's often useful having such snippets ready for debugging and tests.